src/analytics.ts

```diff
@@ -1,5 +1,6 @@
 export const trackEvent =
-  process.env.REACT_APP_GOOGLE_ANALYTICS_ID &&
+  typeof process !== "undefined" &&
+  process?.env?.REACT_APP_GOOGLE_ANALYTICS_ID &&
   typeof window !== "undefined" &&
   window.gtag
```

Comment:
```suggestion
  window?.gtag
```

would that work?

Reply:
Same as above, It would work for client-side but fail on a node where `window` is not declared.